src/tf_mastodon/tf_mastodon_deploy.md

```diff
@@ -0,0 +1,152 @@
+
+It may takes a few minutes until the transfer is successful. We would recommend that you keep your windows open, otherwise you would need to re-login again by using your mnemonic keys. If you accidentally closed your window, you will not have to worry as the transfer will still proceed and you can simply re-login again.
+
+### Generate your SSH Keys
+
+After you have successfully transfered some TFT, you will be asked to generate SSH Keys (for new accounts only) by clicking **‘Generate’** button on the right side. This key is used to authenticate and establish an encrypted communication channel to your Mastodon.
+
+![mnemonics](img/getmastodon_credentials_ssh.png)
+
+If you have previously generated SSH keys on existing profile, you may skip this step.
+Once you have done all the steps above, we can move on to setup your first Mastodon server by clicking either the 'Basic' or 'Advance' setup options.
+
+![mnemonics](img/getmastodon_setup.png)
+
+
+## Setting up Your Mastodon Server
+
+- **Basic** Setup: Mastodon Server deployment setting with pre-configured deployment setup (capacity and network configurations)
+- **Advance** Setup: Mastodon Server deployment setting for users who prefer to choose their own deployment configuration (capacity and network configurations). For this you still need to configure some variables like admin email and deployment name before configuring the advance setup.
+
+## Basic Setup
+
+![](img/getmastodon_basic.png)
+
+Basic Mastodon Server Setup consists of:
+- Mastodon instance name: This will be used to identify your deployment, as well as your Mastodon Server subdomain. Please add a name to your new server.
+- Admin Email: Please add an email used as the server's administrator's email.
+- Region: Please choose the preferred region for new Mastodon server.
+- Enable TF Connect: You can choose of whether you'd like an extra step for your server users to login using TFConnect App or not.
+- Certified: You can choose of whether you'd like to deploy your server using Threefold certified 3Node or DIY node farms. Certified nodes are nodes that farmers bought from ThreeFold Certified Nodes partners. 
+
+If you choose to only do basic setup, you don't need to configure the advance setup. Once you have filled the forms above, simply click **'Deploy'** and you will be notified when your deployment is successful!
+
+![](img/getmastodon_success.png)
+
+Please keep in mind that **you need to have sufficient amount of TFTs** in your deployment wallet in order your deployment to be successful. 
+
+Upon a successful server deployment, you will need to also have sufficient TFTs to keep your deploy alive. Failure to do so will result in server deactivation (not deletion), and You will be given 2 weeks of grace period time to top up your account to reactivate your servers. [Click here for more detailed information about Grace Period.](https://library.threefold.me/info/manual/#/manual__grace_period?id=grace-period).
```

Comment:
Grace period link redirects to the old manual. https://library.threefold.me/info/manual/#/manual__grace_period?id=grace-period

Reply:
ok